client/css/style.css

```diff
@@ -1614,8 +1624,8 @@ kbd {
 	animation-fill-mode: forwards;
 	-webkit-animation-timing-function: ease-in;
 	animation-timing-function: ease-in;
-	-webkit-animation-delay: .4s;
-	animation-delay: .4s;
+	-webkit-animation-delay: .1s;
+	animation-delay: .1s;
```

Comment:
> Why?

To keep all our tooltips consistent, and this is not negotiable 😅

> You're asking to break this, not fix.

*Sigh*
No, I'm not. How is keeping reasonable tooltip *delays* (again, **not** animation durations) breaking anything?
I may very well be missing something, but please explain me *why* it is needed to reduce the tooltip apparition **delay**? I honestly have no idea how the animation delay could collide with any other animation out there, but if it did, `.1s` would be more subject to do that than `.4s`.

Anyway, I really don't think there is anything to break here, so unless I'm missing something obvious, please change them back.